scripts/install.sh

```diff
@@ -17,33 +16,26 @@ unameOut="$(uname -s)"
 case "${unameOut}" in
     Linux*)     os=linux;;
     Darwin*)    os=macos;;
-    *)          os="UNKNOWN:${unameOut}"
+    *)
+        echo "Unsupported OS: ${unameOut}"
+        exit 1
 esac
 
-arch=`uname -m`
 url="https://github.com/snyk-labs/helm-snyk/releases/download/${latest_version}/helm-snyk-${os}"
 
-if [ "$url" = "" ]
-then
-    echo "Unsupported OS / architecture: ${os}_${arch}"
-    exit 1
-fi
-
-filename="helm-snyk-${os}"
+curl -sSL -O $url
+curl -sSL -O "${url}.sha256"
 
-if [ -n $(command -v curl) ]
```

Comment:
It's common to [support `wget` in the install script](https://github.com/databus23/helm-diff/blob/master/install-binary.sh#L78-L82)